src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -46,7 +46,7 @@ class ClusterMonitorSupervisor(monitorConfig: MonitorConfig, gdDAO: DataprocDAO,
           logger.error("Error occurred recreating cluster", e)
         }
       } else {
-        logger.warn(s"Received RecreateCluster message for cluster ${cluster} but cluster recreation is disabled.")
+        logger.warn(s"Received RecreateCluster message for cluster ${cluster.googleProject}/${cluster.clusterName} but cluster recreation is disabled.")
```

Comment:
great catch, thank you

Reply:
You're too kind.

Literally - as I now see, this work is only half done (need to extract the strings) and I'll fix that now.